api/go.mod

```diff
@@ -4,16 +4,15 @@ go 1.18
 
 require (
 	github.com/openshift/api v0.0.0-20220124143425-d74727069f6f // release-4.10
-	k8s.io/apimachinery v0.23.5
+	k8s.io/apimachinery v0.25.15
```

Comment:
Is this minor version update needed? Isn't there a `v0.23.x` version with the fix?

Reply:
The oldest release branch with the fix is `v0.25.15`.